### PR TITLE
add LICENSE file #67

### PR DESCRIPTION
**Pull request by cedrusx**
Adding a LICENSE file (Apache 2.0) into the top directory.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2013-2018 Docker, Inc.
```
Comment (schornakj) [inline LICENSE:179]: Need to update the copyright to match the copyright used in the other source files, which should be `Copyright 2018-2020 Southwest Research Institute`.

Comment (cedrusx) [reply LICENSE:179]: Oh this was a huge mistake... Thank goodness you found it.

Comment (schornakj) [reply LICENSE]: To be fair, I think this is the first time I've ever actually scrolled all the way through a `LICENSE.txt` file :)

**Comment by schornakj** [comment]
Looks good to me. @marip8, what do you think?

File: LICENSE (hunk 1 above)
Comment (cedrusx) [reply LICENSE]: I guess it's the first time you ever reviewed a PR dedicted to add a LICENSE :stuck_out_tongue:

**Comment by marip8** [comment]
Looks good to me; @cedrusx thanks for the contribution

**Pull request merged**